src/modules/core/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import ReactSlider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+
+type Appearance = {
+  theme?: 'primary' | 'danger';
+};
+
+interface Props {
+  value: number;
+  max?: number;
+  limit?: number;
+  appearance?: Appearance;
+  onChange: (val: any) => void;
+}
+
+const displayName = 'Slider';
+
+const Slider = ({ value, max = 100, onChange, limit, appearance }: Props) => {
+  const [sliderValue, setSliderValue] = useState<number>(value);
+
```

Comment:
It's pretty straight forward to set up the field, you just need to bring in the `useField` component   from formik

```
  const [, , { setValue }] = useField(name);
```

Note that since we are using a slider, we don't actually have to handle any errors :)